Assets/HoloToolkit-Examples/UX/Icons/AdjustIcon.svg

```diff
@@ -0,0 +1,236 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
```

Comment:
why svg instead of png?

Reply:
Oh I see you included both.

Reply:
Yeah, I added the original files, makes it easier for others to either modify existing icons or add new icons based on these.

Reply:
I think we'll def use TextMeshPro to make the default icon pack for vNEXT.